client/service/src/lib.rs

```diff
@@ -233,6 +244,52 @@ async fn build_network_future<
 						notification.header.number().clone(),
 					);
 				}
+				
+				if permissioned_network {
+					let id = BlockId::hash(client.info().best_hash);
+					let raw_allowlist = match client.storage(&id, &StorageKey(well_known_keys::NODE_ALLOWLIST.to_vec())) {
+						Ok(Some(r)) => r,
+						Ok(None) => { 
+							info!(
```

Comment:
This is no error and not worth to be printed as info.

Use `unwrap_or_default()`